### DISCUSSION TITLE
Screenshots taken using keyboard shortcut disappear

### DISCUSSION THREAD

**Discussion by sysfu** [elementary/screenshot-tool#244] [Q&A] [answered]
when I take screenshots using the CTRL-SHIFT-PrtSc shortcut the images are not saved into the default Pictures\Screenshot folder. If I use just the PrtSc key it correctly saves into the default folder.

going through the previously closed github issues, I keep seeing that this is a gala issue. Does anyone know how to fix this?

**Comment by davidmhewitt** [comment, marked as answer]
<kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>PrtSc</kbd> is the default shortcut to take a screenshot of an area of the screen and save it to the clipboard and **not** a file.

<kbd>Shift</kbd>+<kbd>PrtSc</kbd> is the default shortcut to take a screenshot of an area and save it to a file, or check the shortcut overlay (open with the Super/Windows key by default) for a list of available shortcuts.
